app/api/routes/action_condition.py

```diff
@@ -0,0 +1,157 @@
+from fastapi import APIRouter, Depends
+from sqlmodel.ext.asyncio.session import AsyncSession
+
+from app.api.errors import NotFoundError
+from app.db.database import get_db
+from app.models.action_condition import (
+    ActionCondition,
+    ActionConditionRequest,
+    ActionConditionResponse,
+    ActionConditionUpdateRequest,
+)
+from app.models.action_condition_operator import (
+    ActionConditionOperator,
+    ActionConditionOperatorRequest,
+    ActionConditionOperatorResponse,
+    ActionConditionOperatorUpdateRequest,
+    NewConditionTreeRequest,
+)
+from app.services.action_condition import ActionConditionService
+
+condition_router = APIRouter(prefix="/conditions")
+
+
+@condition_router.post("/condition", status_code=201)
+async def create_action_condition(
+    condition_request: ActionConditionRequest, db: AsyncSession = Depends(get_db)
+):
+    condition = await ActionConditionService.create_condition(
+        condition_request, db, True
+    )
+
+    return ActionConditionResponse.model_validate(condition)
```

Comment:
we can do response_model=ActionConditionResponse, also missing function return type